domains/piyush.json

```
@@ -5,6 +5,6 @@
     "email": "bhangalepiyush@gmail.com"
   },
   "record": {
-    "URL": "https://piyush.codes"
+    "URL": "https://officialpiyush.github.io"
```

@officialpiyush If you want it to be a sub-domain instead of a URL redirection, you can use a CNAME record instead and add the sub-domain to your repo settings.

```suggestion
    "CNAME": "officialpiyush.github.io"
```

Let me know if you want it to keep it as is, I'll merge it in.

---

I would like to keep this as it is :)